storage-stackdriver/pom.xml

```diff
@@ -31,15 +31,8 @@
 
   <dependencies>
     <dependency>
-      <groupId>com.google.api.grpc</groupId>
-      <artifactId>grpc-google-devtools-cloudtrace-v1</artifactId>
-      <version>${grpc-google-devtools-cloudtrace-v1.version}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>io.grpc</groupId>
-          <artifactId>grpc-all</artifactId>
-        </exclusion>
-      </exclusions>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>zipkin-translation-stackdriver</artifactId>
```

Comment:
note: later we will scrub this dep in the collector as it will eventually become problematic when we do multiple stackdriver versions.